### PR TITLE
HLR: Gate Users with No Address

### PR DESCRIPTION
**Pull request by ndsprinkle**
## Description
This adds gating to HLR for users without an address in their profile. Merging this is pending content approval from design.

## Screenshots
image

## Acceptance criteria
- [ ] HLR gated for missing addresses

### CODE REVIEW

**Comment by ndsprinkle** [comment]
Confirmed with Christian that the displayed message meets expectations.

File: src/applications/disability-benefits/996/content/contestableIssueAlerts.jsx [hunk 1]
```diff
@@ -66,3 +66,22 @@ export const showWorkInProgress = (
...
+
+export const showHasEmptyAddress = (
+  <AlertBox
+    status="info"
+    headline="You need an address on file to apply"
+    content={
+      <>
+        <p>
+          You need a valid address on file in your profile to apply for higher
+          level reivew.
```
Comment (Mottie) [inline src/applications/disability-benefits/996/content/contestableIssueAlerts.jsx:78]: "Higher-Level Review" needs to be capitalized and include a hyphen.

**Review by Mottie — APPROVED** [review]

**Comment by ndsprinkle** [comment]
Hey @peggygannon , does the above alert message look good? Other than the change Robin pointed out regarding changing "higher level review" to "Higher-Level Review"

**Comment by peggygannon** [comment]
@ndsprinkle @Mottie  a followup question on this. Will the address be updated automatically?

 or should the message included language letting the users know that they should allow time for the address to update in the system before coming back to apply for a HLR?

**Comment by Mottie** [comment]
> Will the address be updated automatically?

Yes, once the profile is updated, it will dynamically check and show the updated content on the veteran info page.

**Comment by peggygannon** [comment]
@Mottie @ndsprinkle  Ok, thank you. 

Here's my recommendation: 

**You need to have an address on file** 
To request a Higher-Level Review, you need to have an address in your VA profile. To add an address, [go to your profile].

**Pull request merged**